src/engine/objects/resolvers/visitors.rs

```diff
@@ -400,7 +455,7 @@ where
     }
 }
 
-pub(super) fn visit_node_query_input<'a, T>(
+pub(crate) fn visit_node_query_input<'a, T: ?Sized, RequestCtx>(
```

Comment:
Why do we need to remove the Sized bound?

Reply:
This one I spent a long time fighting the compiler and this was the only way to make it work. Maybe tomorrow we can hop on for a bit and talk through any possible alternatives.

Reply:
My guess is that if you can change from a `transaction: &'a mut dyn Transaction<RequestCtx>` to a concrete type that implements Transaction, it will probably solve your sizing problem, here.  (Or we'll find out I need to learn rust better.)